pkg/dashboard/helpers.go

```diff
@@ -15,16 +15,26 @@
 package dashboard
 
 import (
+	"fmt"
+
 	uuid "github.com/satori/go.uuid"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
-func printResource(quant resource.Quantity) string {
+func printResource(quant resource.Quantity, resourceType corev1.ResourceName) string {
+	// weird thing you can't look at
 	if quant.IsZero() {
 		return "Not Set"
 	}
-	return quant.String()
+	switch resourceType {
```

Comment:
Minor nit: I think this switch duplicates `quant.String()`. This might be simpler:
```go
if resourceType == corev1.ResourceMemory {
  quant = quant.RoundUp(resource.Mega)
}
return quant.String()
```

Reply:
I am not sure I understand this, but happy to talk about it.

Reply:
Interesting. We were following the pattern from another example. Will that guarantee that `Mi` is returned rather than `k` for the units?  I am not familiar with the quant.RoundUp function.

Reply:
Also, what happens to CPU or other types of values in that case?

Reply:
Not entirely sure why this doesn't want to work, but this is the error I got when I tried it.
`cannot use quant.RoundUp(resource.Mega) (type bool) as type resource.Quantity in assignment go`
Something with a type difference?

Reply:
Decided to just leave this for now. We can optimize later if we want